dataplane/forwarding/fwd.go

```diff
@@ -922,3 +877,23 @@ func (e *Server) Operation(stream fwdpb.Forwarding_OperationServer) error {
 		}
 	}
 }
+
+// NotifySubscribe subscribes to notification for a forwarding context.
+// TODO: Only one notification client is supported per forwarding context.
+func (e *Server) NotifySubscribe(sub *fwdpb.NotifySubscribeRequest, srv fwdpb.Forwarding_NotifySubscribeServer) error {
+	eventCh := make(chan *fwdpb.EventDesc)
+	fn := func(ed *fwdpb.EventDesc) {
+		eventCh <- ed
+	}
+
+	if err := e.UpdateNotification(sub.GetContext(), fn, "callback"); err != nil {
```

Comment:
I guess the notification address is not useful anymore? Maybe a TODO comment to remove confusion?